Tests/Pcap++Test/Tests/FileTests.cpp

```diff
@@ -751,3 +751,40 @@ PTF_TEST_CASE(TestPcapFileReadLinkTypeIPv4)
 
 } // TestPcapFileReadLinkTypeIPv4
 
+PTF_TEST_CASE(TestSolarisSnoopFileRead)
+{
+	pcpp::SnoopFileReaderDevice readerDev(EXAMPLE_SOLARIS_SNOOP);
+	PTF_ASSERT_TRUE(readerDev.open());
+	pcpp::RawPacket rawPacket;
+	int packetCount = 0;
+	int ethCount = 0;
+	int ipCount = 0;
+	int tcpCount = 0;
+	int udpCount = 0;
+	while (readerDev.getNextPacket(rawPacket))
```

Comment:
We should probably also check the timestamp of at least one packet

Reply:
Resolved in 1e32b15